src/extensions/surveys/surveys-extension-utils.tsx

```diff
@@ -540,10 +540,10 @@ const getSurveyInteractionProperty = (survey: Survey, action: string): string =>
     return surveyProperty
 }
 
-export const hasWaitPeriodPassed = (
-    lastSeenSurveyDate: string | null,
-    waitPeriodInDays: number | undefined
-): boolean => {
+const LAST_SEEN_SURVEY_DATE_KEY = 'lastSeenSurveyDate'
+
+export const hasWaitPeriodPassed = (waitPeriodInDays: number | undefined): boolean => {
+    const lastSeenSurveyDate = localStorage.getItem(LAST_SEEN_SURVEY_DATE_KEY)
```

Comment:
i moved the part to get the localStorage value to be inside `hasWaitPeriodPassed` as it's the only place it's being used, so no need to pass it as a parameter and we can just get it directly